### PR TITLE
Add python documentation

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [ ] Passes the tests
- [ ] HOPSWORKS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: docs/compute/python.md [hunk 1]
```diff
@@ -1 +1,151 @@
-# Python docs
\ No newline at end of file
+# Python environment
+
+Hopsworks provides a premade environment for data engineering, machine learning and more general data science development.
```
Comment (tkakantousis) [inline docs/compute/python.md:3]:
```suggestion
Hopsworks comes out of the box with a Python environment for data engineering, machine learning and more general data science development.
```

It's fine to keep the current text, just thought of a suggestion to make the text more context-specific

File: docs/compute/python.md [hunk 2]
```diff
@@ -1 +1,151 @@
...
+Managing the project's Python environment is provided as a service on Hopsworks and can be found in the `Python libraries` page under the `Project settings` menu.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/navigate.gif" alt="Open the Python libraries page">
+    <figcaption>Open the Python libraries page</figcaption>
+  </figure>
+</p>
+
+When a project is created, the python 3.8 environment is automatically enabled. So no manual steps are required to start developing your machine learning application.
```
Comment (tkakantousis) [inline docs/compute/python.md:20]:
```suggestion
When a project is created, the python 3.8 environment is automatically enabled. So no additional steps are required to start developing your machine learning application.
```

File: docs/compute/python.md [hunk 3]
```diff
@@ -1 +1,151 @@
...
+
+##Listing installed libraries
+
+The list of installed libraries is displayed on the page. It is possible to filter based on the name and package manager to find a particular library.
```
Comment (tkakantousis) [inline docs/compute/python.md:24]:
```suggestion
The list of installed libraries is displayed on the page. It is possible to filter based on the name and package manager to search for a particular library.
```

File: docs/compute/python.md [hunk 4]
```diff
@@ -1 +1,151 @@
...
+
+##Installing libraries
+
+Python packages can be installed from the following sources:
+
+* PyPi, using pip package manager
+* A conda channel, using conda package manager
+* Packages saved in certain file formats, currently we support .whl or .egg
+* A public or private git repository
+* A requirements.txt file to install many libraries at the same time using pip
```
Comment (tkakantousis) [inline docs/compute/python.md:41]:
```suggestion
* A requirements.txt file to install multiple libraries at the same time using pip
```

File: docs/compute/python.md [hunk 5]
```diff
@@ -1 +1,151 @@
...
+* An environment.yml file to install many libraries at the same time using conda and pip
```
Comment (tkakantousis) [inline docs/compute/python.md:42]:
```suggestion
* An environment.yml file to install multiple libraries at the same time using conda and pip
```

File: docs/compute/python.md [hunk 6]
```diff
@@ -1 +1,151 @@
...
+
+###Install by name and version
+
+Enter the name and optionally the desired version to install.
```
Comment (tkakantousis) [inline docs/compute/python.md:46]:
```suggestion
Enter the name and, optionally, the desired version to install.
```

File: docs/compute/python.md [hunk 7]
```diff
@@ -1 +1,151 @@
...
+    <figcaption>Installing library from file</figcaption>
+  </figure>
+</p>
+
+###Install from .git repository
+
+To install from a git repository simply provide the repository URL. The URL you should provide is the same as you would enter on the command line using `pip install git+{repo_url}`.
+In the case of a private git repository, also select whether it is a GitHub or GitLab repository and the preconfigured access token for the repository.
+
+**Note**: If you are installing from a git repository which is not GitHub or GitLab simply supply the access token in the URL. Keep in mind that in this case the access token may be visible in logs for other users in the same project to see.
```
Comment (tkakantousis) [inline docs/compute/python.md:89]:
```suggestion
**Note**: If you are installing from a git repository which is not GitHub or GitLab simply supply the access token in the URL. Keep in mind that in this case the access token may be visible to all users tin the project\s in logs.
```

File: docs/compute/python.md [hunk 8]
```diff
@@ -1 +1,151 @@
...
+
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/conflicts.png" alt="Conflicts detected in environment">
+    <figcaption>Conflicts detected in environment</figcaption>
+  </figure>
+</p>
+
+##Recreating environment
```
Comment (tkakantousis) [inline docs/compute/python.md:122]:
```suggestion
##Recreating the environment
```

File: docs/compute/python.md [hunk 9]
```diff
@@ -1 +1,151 @@
...
+After removing the environment, simply recreate it by clicking `Create Environment`.
+
+<p align="center">
+  <figure>
+    <img src="../../assets/images/python/create_env.png" alt="Create environment">
+    <figcaption>Create environment</figcaption>
+  </figure>
+</p>
+
+##Exporting an environment
```
Comment (tkakantousis) [inline docs/compute/python.md:142]: Do we support importing a yml file in the new UI? Also I would move this section before the `Remove environment`.

**Review by tkakantousis — APPROVED** [review]

File: docs/compute/python.md (hunk 9 above)
Comment (robzor92) [reply docs/compute/python.md:142]: I moved the Export before the Remove environment. There is no option to import an environment.yml and create an environment from that at the moment, it also came to my knowledge recently. You can however install an environment.yml or requirements.txt on top of the default environment right now.

**Pull request merged**